### PR TITLE
Upgrade material ui to v4

### PR DESCRIPTION
**Pull request by kdjohnson**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { connect } from 'react-redux'
 import { fetch_advising } from './actions/advisingActions'
 import { fetch_terms } from './actions/termsActions'
 import { withStyles } from '@material-ui/core/styles'
+// import { makeStyles } from '@material-ui/styles'
```
Comment (amgoodfellow) [inline src/App.js]: We can probably just delete that commented out guy

```suggestion
```

File: src/components/CourseDetails.js [hunk 2]
```diff
@@ -78,7 +79,7 @@ class CourseDetails extends React.Component {
           </Button>
 
           <Dialog
-            role="dialog"
+            //role="dialog"
```
Comment (amgoodfellow) [inline src/components/CourseDetails.js]:
```suggestion
```

File: src/index.js [hunk 3]
```diff
@@ -32,17 +30,14 @@ const theme = createMuiTheme({
 })
 
 const root_element = 'root'
-const jss = create(jssPreset())
 
 ReactDOM.render(
   <I18nextProvider i18n={i18n}>
-    <JssProvider jss={jss}>
-      <MuiThemeProvider theme={theme}>
+      <ThemeProvider theme={theme}>
```
Comment (amgoodfellow) [inline src/index.js]: Everything looks right in here theme-wise. Unfortunately, the calendar is now back to its default color. This makes me very sad :( 

I guess I'll have to go update that project sometime...

**Review by amgoodfellow — COMMENTED** [review]
Aside from a couple commented things that could be deleted, everything looks good to me. Courses still runs. We'll just have to remember to update the calendar when we accept this change at OU

**Review by amgoodfellow — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (kdjohnson) [reply src/App.js]: Hmm, yea I definitely should have refactored before pushing but I figured that would be better of in another PR I'm working on as it moves over to using functions.

**Pull request merged**